yt/python/yt/environment/yt_env.py

```diff
@@ -609,6 +615,10 @@ def start(self, on_sequoia_started_func=None, on_masters_started_func=None):
             if self.yt_config.replicated_table_tracker_count > 0:
                 self.start_replicated_table_trackers(sync=False)
 
+            if self.yt_config.enable_auth:
+                # The following `synchroize` might use http client, so the user has to be created before that.
+                self.create_admin_user()
```

Comment:
I think in this point the cluster might not be started (synchronized) yet, so this calls may fail. I think that it should be done _after_ starting masters and initial synchronization, but _before_ starting http proxies

Reply:
This means that http proxies should be started after the initial synchronization, but this synchronization itself requires http proxies - unless we start bootstrapping the cluster using native client only, no matter what the value of `use_native_client` is. 

BTW, if we make `create_admin_user` optional, we will face the same issue - http client won't be able to be used as  `cluster_client`. 

Are we going to always use native client as `cluster_client`?